frontend/packages/helm-plugin/src/actions/providers.ts

```diff
@@ -16,3 +19,27 @@ export const useHelmActionProvider = (scope: HelmActionsScope) => {
 
   return [actions, true, undefined];
 };
+
+export const useHelmActionProviderForTopology = (element: GraphElement) => {
+  const nodeType = element.getType();
+  const scope = React.useMemo(() => {
+    const releaseName = element.getLabel();
+    const { namespace } = getResource(element as Node).metadata;
+    return {
+      releaseName,
+      namespace,
+      actionOrigin: 'topology',
+    };
+  }, [element]);
+  const { t } = useTranslation();
+  const actions = React.useMemo(
+    () => [
+      getHelmUpgradeAction(scope, t),
+      getHelmRollbackAction(scope, t),
+      getHelmDeleteAction(scope, t),
+    ],
+    [scope, t],
+  );
```

Comment:
We can actually re use the original provider.
```suggestion
  const actions = useHelmActionProvider(scope);
```

Reply:
@rohitkrai03 fixed!